client/platform/web-girder/views/NavigationBar.vue

```diff
@@ -42,49 +42,59 @@ export default {
 </script>
 
 <template>
-  <v-app-bar app>
-    <NavigationTitle :name="brandData.name" />
-    <v-tabs
-      icons-and-text
-      color="accent"
-    >
-      <v-tab
-        exact
-        @click="route(location)"
+  <div>
+    <v-app-bar app>
+      <NavigationTitle :name="brandData.name" />
+      <v-tabs
+        icons-and-text
+        color="accent"
       >
-        Data
-        <v-icon>mdi-database</v-icon>
-      </v-tab>
-      <JobsTab />
-    </v-tabs>
-    <v-spacer />
-    <GirderSearch
-      :search-types="['user', 'folder']"
-      placeholder="search"
-      hide-options-menu
-      hide-search-icon
-      class="mx-2 grow"
-      @select="route"
-    />
-    <v-btn
-      text
-      :to="{ name: 'summary' }"
-    >
-      <v-icon class="pr-2">
-        mdi-format-list-bulleted-square
-      </v-icon>
-      Stats
-    </v-btn>
-    <user-guide-button />
-    <v-btn
+        <v-tab
+          exact
+          @click="route(location)"
+        >
+          Data
+          <v-icon>mdi-database</v-icon>
+        </v-tab>
+        <JobsTab />
+      </v-tabs>
+      <v-spacer />
+      <GirderSearch
+        :search-types="['user', 'folder']"
+        placeholder="search"
+        hide-options-menu
+        hide-search-icon
+        class="mx-2 grow"
+        @select="route"
+      />
+      <v-btn
+        text
+        :to="{ name: 'summary' }"
+      >
+        <v-icon class="pr-2">
+          mdi-format-list-bulleted-square
+        </v-icon>
+        Stats
+      </v-btn>
+      <user-guide-button />
+      <v-btn
+        text
+        @click="logout"
+      >
+        Logout
+      </v-btn>
+    </v-app-bar>
+    <v-alert
+      v-if="brandData.alertMessage"
+      type="info"
```

Comment:
Let's unify the color and icon across all places this alert may appear to warning yellow?

`type=warning`

Also, I know `v-banner` is specifically designed to fit below a `v-app-bar`.  Is there any difference between these, such as how the page layout is impacted?

Reply:
I am not easily finding an answer to the page layout impact. I suspect that since it looks like Banner was created after Alert that this might be an added feature. I am fine with going with the component known to work with our layout and will change it to `v-banner`